l10n_jp_summary_invoice_sale_partner/reports/report_summary_invoice_templates.xml

```diff
@@ -0,0 +1,20 @@
+<odoo>
+    <template
+        id="bill_line_section_layout_inherit"
+        inherit_id="l10n_jp_summary_invoice.bill_line_section_layout"
+    >
+        <xpath expr="//tr[td/span[@t-field='invoice.invoice_date']]" position="after">
+            <t
+                t-set="sale_partner"
+                t-value="invoice.get_summary_invoice_sale_partner()"
+            />
+            <t t-if="sale_partner">
+                <tr>
+                    <td t-att-colspan="line_columns">
+                        <span t-field="sale_partner.name" />
```

Comment:
We may add a label, not only the field value. We should perhaps remove `.name` here.